packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -109,6 +111,26 @@ class Scrollbar extends StatelessWidget {
   /// [showTrackOnHover] can be replaced by this and will be deprecated.
   final bool? trackVisibility;
 
+  /// Controls the button visibility.
+  ///
+  /// If this property is null, then [ScrollbarThemeData.buttonVisibility] of
+  /// [ThemeData.scrollbarTheme] is used. If that is also null, the default value
+  /// is false.
+  final bool? buttonVisibility;
+
+  /// Controls the presentation style of the scrollbar button.
+  ///
+  /// If this property is null, then [ScrollbarThemeData.buttonStyles] of
+  /// [ThemeData.scrollbarTheme] is used. If that is also null, the default value
+  /// is [ScrollbarButtonStateStyles].
+  ///
+  /// See also:
+  ///
+  ///  * [ScrollbarButtonStateStyles], a scrollbar button style that use
+  ///  [ScrollbarButtonStateColors] which the default value follows the
+  ///  Windows platform style.
+  final ScrollbarButtonStyles? buttonStyles;
```

Comment:
Can we make this class and any references singular? Otherwise it seems like a list of styles, or more than one style.

```suggestion
  final ScrollbarButtonStyle? buttonStyle;
```

Reply:
There may be some conflict here with the ButtonStyle class in the material library. That may get a bit confusing, _**Or**_ we might be able use it here... please do check it out and see if maybe the ScrollbarButtonStyle can be a ButtonStyle, or if the ScrollbarButtonStyle could inherit from it.